src/main/java/org/terasology/dynamicCities/settlements/SettlementEntityManager.java

```diff
@@ -251,10 +242,34 @@ private EntityRef createSettlement(EntityRef siteRegion) {
 
         //add region entities
         getSurroundingRegions(settlementEntity);
+        settlementEntity.saveComponent(regionEntitiesComponent);
 
         //Hook into the economy module's MarketUpdaterSystem
         settlementEntity.send(new SubscriberRegistrationEvent());
 
+
+        /* Add the watched chunks to the settlement */
+
+        //Convert the settlement radius into chunks
+        int settlementRadius = SettlementConstants.SETTLEMENT_RADIUS;
+        int chunkWidth = CHUNK_SIZE.x;
```

Comment:
Yeah, that's a good point. I'm not sure about using min, because then the resulting area would be an ellipse if x and z are different, with more chunks being watched than are needed. I'll see if there's a better way, such as calculating the x and z distance separately, or using the actual block distance.